### ISSUE TITLE
polygon tool: double-click too sensitive

### ISSUE THREAD

**Issue by bertsky** [OCR4all/LAREX#322]
One of the most used tools is probably the polygon drawer. However, there are some practical nuisances here IMO:

- there is no way to "undo" the last vertex
- there is no way to cancel the final closure
- the time and distance for the double click rule are way too sensitive

(In effect, I often have to redo long polgons simply because LAREX thought the next vertex was the final one.)

**Comment by maxnth** [comment]
I agree, the default values time / distance delta values for double clicks were a bit funky, I lowered them significantly and users are now able to set these values themselves starting with a0e6e69.
